qa-test

```diff
@@ -17,6 +17,10 @@ test_python()
   find . -path ./api -prune -o -name '*.py' -exec echo "Running pylint on {}" \; -exec ${pylintCmd} -E --disable=C,R --init-hook="import sys; sys.path.append('./tools')" --persistent=n {} \;
 }
 
+test_typos()
+{
+  typos --exclude builder* --exclude tree/20_cfe_basics/*
```

Comment:
we should include the 20_cfe_basics folder but exclude the cfengine subdirectory